src/tests/end2end/CopyTests.cpp

```diff
@@ -355,6 +368,36 @@ TEST_P(CopyTests_T2B, RowPitchUnaligned) {
     }
 }
 
+// Test that copying regions of each texture 2D array layer works
+TEST_P(CopyTests_T2B, Texture2DArrayRegion)
+{
+    // TODO(jiawei.shao@intel.com): support 2D array texture on OpenGL, D3D12 and Metal.
```

Comment:
In a follow-up patch could we have a DAWN_SKIP_TEST_IF like ANGLE_SKIP_TEST_IF?

Reply:
Ack